packages/shorebird_cli/lib/src/code_push_client_wrapper.dart

```diff
@@ -412,6 +419,8 @@ ${archMetadata.arch} artifact already exists, continuing...''',
       }
     }
 
+    const baseLogMessage = 'Creating aab artifact';
```

Comment:
Seems a bit misleading to call this `creating` because the upload will likely account for majority of the time spent.

Reply:
Yep, log messages need to be updated